src/content/en/2019/accessibility.md

```diff
@@ -181,61 +183,61 @@ Shortcut keys set via the [`aria-keyshortcuts`](https://www.w3.org/TR/wai-aria-1
+  <div id="fig8-caption" class="visually-hidden">Figure 8. Vertical bar chart titled "Percentage of pages using each ARIA attribute" displaying percentage data, ranging from 0 to 25 in increments of 5, vs. bars representing each attribute. Aria-hidden: 23.46%, aria-label: 17.67%, aria-expanded: 8.68%, aria-current: 7.76%, aria-labelledby: 6.85%, aria-controls: 3.56%, aria-haspopup: 2.62%, aria-invalid: 2.68%, aria-describedby: 1.69%, aria-live: 1.04%, aria-required: 1%.</div>
+  <figcaption>Figure 8. Percentage of pages using each ARIA attribute.</figcaption>
+</figure>
 
 #### The `role` attribute
 
 The "role" attribute is the most important attribute in the entire ARIA specification. It's used to inform the browser what the purpose of a given HTML element is (i.e., the semantic meaning). For example, a `<div>` element, visually styled as a button using CSS, should be given the ARIA role of `button`.
 
-Currently, **46.91% of pages** use at least one ARIA role attribute. In Figure 9 below, we’ve compiled a list of the top ten most widely used ARIA role values.
-
-```<insert graphic of metric 09_05>```
-
-*Figure 9. Top 10 aria roles*
+Currently, 46.91% of pages use at least one ARIA role attribute. In Figure 9 below, we’ve compiled a list of the top ten most widely used ARIA role values.
 
-Alt text: Figure 9. Horizontal bar chart titled "percent of pages using role" with bars for each role type vs percent of sites using ranging from 0 to 25 in increments of 5. Navigation: 20.4%; search: 15.49%; main: 14.39%; banner: 13.62%; contentinfo: 11.23%; button: 10.59%; dialog: 7.87%; complementary: 6.06%; menu: 4.71%; form: 3.75%
+<figure>
+  <iframe aria-labelledby="fig9-caption" width="600" height="370" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=176877741&format=interactive"></iframe>
+  <div id="fig9-caption" class="visually-hidden">Figure 9. Horizontal bar chart titled "top 10 most used ARIA roles" with bars for each role type vs percent of sites using ranging from 0 to 25 in increments of 5. Navigation: 20.4%; search: 15.49%; main: 14.39%; banner: 13.62%; contentinfo: 11.23%; button: 10.59%; dialog: 7.87%; complementary: 6.06%; menu: 4.71%; form: 3.75%</div>
+  <figcaption>Figure 9. Top 10 most used ARIA roles.</figcaption>
+</figure>
 
 Looking at the results in Figure 9, we can draw two surprising conclusions:
 
 ##### Updating UI frameworks could be the way forward for accessibility across the web
 
 The top 5 roles, all appearing on 11% of pages or more, are landmark roles. These are used to aid navigation—not to describe the functionality of a widget, such as a combo box. This is a surprising result because the main motivator behind the development of ARIA was to give web developers the capability to describe the functionality of widgets made of generic HTML elements (like a `<div>`).
```

Comment:
```suggestion
The top 5 roles, all appearing on 11% of pages or more, are landmark roles. These are used to aid navigation, not to describe the functionality of a widget, such as a combo box. This is a surprising result because the main motivator behind the development of ARIA was to give web developers the capability to describe the functionality of widgets made of generic HTML elements (like a `<div>`).
```